pkg/controller/certificatemanager/certificatemanager_test.go

```diff
@@ -484,15 +538,14 @@ var _ = Describe("Test CertificateManagement suite", func() {
 			By("counting the number of pem blocks in the configmap")
 			bundle := configMap.Data[certificatemanagement.TrustedCertConfigMapKeyName]
 			numBlocks := strings.Count(bundle, "certificate name:")
-			// While we have the ca + 4 certs, we expect 3 cert blocks (+ 2):
+			// While we have the ca + 4 certs, we expect just 2 cert blocks (+ 2):
```

Comment:
Have you considered adding a 5th cert when making the bundle, so we cover both the modern legacy secret use case and the legacy usage legacy secret?